lib/repositories/switch-repository.ts

```diff
@@ -61,7 +62,7 @@ export class SwitchRepository implements BaseSwitchRepository {
     );
 
     SwitchRepository.log.info({ res: result.Item }, 'get result');
-    if (result.Item && result.Item.lower <= amount) {
+    if (result.Item && BigNumber.from(result.Item.lower).lte(amount)) {
```

Comment:
```suggestion
    if (result.Item && BigNumber.from(result.Item.lower).lte(BigNumber.from(amount))) {
```
? because amount is type `string` ?

Reply:
oh actually looking at your test guess we don't need it